index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>collabortion project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p id="tribute-info">The man who saved a billion lives</p>
+        
+        <a href="#" id="tribute-link" target="_blank"></a>
```

Comment:
I would recommend adding a div here for better flexibility when using CSS 👍

Reply:
- [ ] Don't you think a recommendation should be given with a reason? If I recommend you to do something without a reason, would you think of doing it? Yes, I am also recommending you to give a reason for your recommendation BECAUSE a recommendation should explain the WHY. 😄

Reply:
I already explained the reason for better flexibility when using CSS

Reply:
I would recommend adding a div here for better flexibility when using CSS. 👍
for they will be children of a parent container and when you start using a display feature in the container (parent) it will be applied for all of its items (children).